packages/backend/apps/multitenancy/constants.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class TenantType(models.TextChoices):
+    SIGN_UP = "sign_up", "Sign Up"
```

Comment:
Do you have a better idea how to name this? 
I'm kind of afraid using a verb as a noun is going to be confusing for all people.

I was confused even though I'm pretty sure I suggested the name :D 

(Can't be `default` either IMO)

Reply:
sure, tbh I was thinking about `default` but your suggestion make me think that maybe `sign_up` will be better name :D

Reply:
But `default` has its own problems. Could be interpreted as `default` selection when logging in.

Reply:
Anyway, I checked and Netlify calls it `default` as well